### PR TITLE
Added `!eval` and `!mute`, merged `!purge` into moderation.py

### PR DESCRIPTION
**Pull request by Kronifer**
Pretty much the title

### CODE REVIEW

**Comment by Kronifer** [comment]
since replit sucks ass, we will remove pycache files aald

**Comment by dannynotsmart** [comment]
yes replit bad

**Review by hypergamer80 — APPROVED** [review]
everything works since you tested it all in the dsc server, so lgtm

**Comment by Kronifer** [comment]
Awesome

**Review by AbooMinister25 — APPROVED** [review]
Alright, so this looks decent, eval looks nice. The tio integration looks good, and overall, it seems decent.

**Comment by hypergamer80** [comment]
will merge after dawnofmidnight gives his review.

File: bot/constants.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
 EXTENSIONS = pathlib.Path("bot/exts/")
 BOT_REPO_URL = "https://github.com/cat-dev-group/bot"
 
+MESSAGE_LIMIT = 2000
```
Comment (dawnofmidnight) [inline bot/constants.py:14]: Message limit is defined in config,yml. Do we want to pull from there yet, or should we wait till we have a function that makes it simple?

File: bot/constants.py [hunk 2]
```diff
@@ -11,6 +11,9 @@
...
+
 
 class Channels:
     devlog = 809125747037306960
+    modlog = 834500035982786620
```
Comment (dawnofmidnight) [inline bot/constants.py:19]: Same thing as with message limit.,

File: config.yml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 bot:
-  prefix:      "!"
+  prefix:      "c!"
```
Comment (dawnofmidnight) [inline config.yml:2]: This needs to be changed back to `!`, does it not? I thought `c!` was simply for testing.

File: bot/constants.py (hunk 1 above)
Comment (Kronifer) [reply bot/constants.py:14]: We don't use this yet, I'll delete in a cleanup pr

File: config.yml (hunk 3 above)
Comment (Kronifer) [reply config.yml:2]: Alright, I'll change that soon

File: bot/constants.py (hunk 2 above)
Comment (Kronifer) [reply bot/constants.py:19]: This should be pulled from config, I'll check

File: bot/constants.py (hunk 1 above)
Comment (dawnofmidnight) [reply bot/constants.py:14]: I can deal with config stuff. I'll open a pr soon.

Comment (Kronifer) [reply bot/constants.py:14]: Do you mind if I merge these now and address pulling from config in a later pr?

Comment (dawnofmidnight) [reply bot/constants.py:14]: Yes, that's what I meant. I can merge.

**Review by dawnofmidnight — APPROVED** [review]

File: config.yml (hunk 3 above)
Comment (hypergamer80) [reply config.yml:2]: i agree, we need a better prefix

Comment (Kronifer) [reply config.yml:2]: I suggest &, would that work?

**Pull request merged**